src/lib/app-helper.js

```diff
@@ -516,6 +517,32 @@ function deleteUserConfig (configData) {
   fs.writeFileSync(configData.file, yaml.safeDump(phyConfig))
 }
 
+/** Get ALL Extension Point List for the selected org
+ *
+ * @param consoleCLI
+ */
+async function getAllExtensionPoints (consoleCLI) {
+  const projectConfig = aioConfig.get('project')
```

Comment:
can you pass the aioConfig as param, it's available under the fullConfig.aio, that way we keep the loading of aioConfig inside the config-loader, that makes it easier for testing/mocking too

Reply:
I have changed the function interface to accept orgId( don't need entire config anyway)